.gitignore

```diff
@@ -146,3 +146,8 @@ npm-debug.log*
 static/dist
 derby.log
 metastore_db
+
+airflow-*.err
```

Comment:
Are these meant to be here?

Reply:
Yeah. When you run airflow integration tests as unit tests airflow will write those airflow* logs in the main directory. I think it's good to .gitignore them so that they are not added accidentally.